.github/workflows/publish-internal.yml

```diff
@@ -90,5 +96,7 @@ jobs:
             --output text \
             --query repositoryEndpoint)
 
+          hatch build --clean
```

Comment:
I moved these here because they seemed more closely related to `hatch publish` than manually mucking with the version and `pyproject.toml`.